src/app/shared/shared.module.js

```diff
@@ -29,6 +30,7 @@ export default angular.module('sharedModule', [])
     .factory('openWeatherMapsService', openWeatherMapsService)
     .factory('geoNamesService', geoNamesService)
     .factory('routingFunctionsService', routingFunctionsService)
+    .factory('mailing', mailing)
```

Comment:
Didn't you decide to use services instead of factories?
And dont forget to set a more descriptive name like 'MailingService' here too

Reply:
Already solved when updating from development and solving conflicts!